### PR TITLE
Create separate route for Templates

### PR DESCRIPTION
**Pull request by imanmoreira**
- separated the routes for the templates page
- Making different colored buttons common components

### CODE REVIEW

File: frontend/src/advising/Templates/TemplateListPage.tsx [hunk 1]
```diff
@@ -188,7 +187,7 @@ const TemplatesList = ({ searchQuery }: TemplatesListProps) => {
   }, [searchQuery]);
 
   return (
-    <TemplatesContainer>
+    <>
```
Comment (imanmoreira) [inline frontend/src/advising/Templates/TemplateListPage.tsx:190]: This is to get rid of the weird double red outline display that was happening on the TemplateListPage

**Comment by NilsBacke** [comment]
[Move all templates pages to be their own route (templates list, new template)](https://trello.com/c/ZwEvRNrY/277-move-all-templates-pages-to-be-their-own-route-templates-list-new-template)

File: frontend/src/home/Home.tsx [hunk 2]
```diff
@@ -132,16 +133,6 @@ const LogoutButton = styled.div`
   margin-right: 8px !important;
 `;
 
-const ColorButton = withStyles((theme: Theme) => ({
```
Comment (NilsBacke) [inline frontend/src/home/Home.tsx:135]: I believe @andrewjleung was also working on similar button refactors. What we have here looks good, just @'ing to make him aware.

**Review by NilsBacke — APPROVED** [review]

File: frontend/src/home/Home.tsx (hunk 2 above)
Comment (andrewjleung) [reply frontend/src/home/Home.tsx:135]: @justineluo and I ended up leaving out button refactors, so this should be all fine 👍

**Review by andrewjleung — APPROVED** [review]

**Pull request merged**